internal/providers/outscale_oapi/outscale_oapi.go

```diff
@@ -545,14 +545,18 @@ func (provider *OutscaleOAPI) readVolumes() ([]Object, error) {
 	read, err := provider.client.ReadVolumes(context.Background(), osc.ReadVolumesRequest{
 		Filters: &osc.FiltersVolume{
 			VolumeStates: &[]string{
-				"creating", "available", "in-use", "error", // skip deleting
+				"creating", "available", "in-use", "error",
 			},
 		},
 	})
 	if err != nil {
 		return nil, fmt.Errorf("read volumes: %w", getErrorInfo(err))
 	}
 	for _, volume := range *read.Volumes {
+		// When a volume was created from a snapshot, it will appear as "deleting" if filtered by it's state
```

Comment:
Not very clear...

Proposal: When a deleting volume was created from a snapshot, it will be returned even if deleting is missing from `Filters.VolumeStates`